### PR TITLE
ML-707: bug fix in multiple integer key columns

### PR DESCRIPTION
**Pull request by katyakats**

### CODE REVIEW

**Review by dinal — APPROVED** [review]

File: storey/utils.py [hunk 1]
```diff
@@ -164,6 +164,7 @@ def update_in(obj, key, value):
 
 
 def hash_list(list_to_hash):
+    list_to_hash = [str(element) for element in list_to_hash]
```
Comment (gtopper) [inline storey/utils.py:167]: Looks like this is actually the [fastest way](https://waymoot.org/home/python_string/) to do it. ✅

File: integration/test_aggregation_integration.py [hunk 2]
```diff
@@ -1363,3 +1363,46 @@ def test_non_existing_key_query_by_key_from_v3io_key_is_list(setup_teardown_test
     controller.emit({'nameeeee': 'katya'}, 'katya')
     controller.terminate()
     controller.await_termination()
+
+
+def test_multiple_keys_int(setup_teardown_test):
```
Comment (gtopper) [inline integration/test_aggregation_integration.py:1368]: Maybe mention in a comment that this is regression for ML-707.

Comment (gtopper) [reply integration/test_aggregation_integration.py:1368]: I don't think we need an integration test to test this, a unit test would suffice.

**Review by gtopper — APPROVED** [review]
Great (see nonblocking comments).

**Pull request merged**